### DISCUSSION TITLE
I'm trying to apply for the Github Student Developer Pack, but i can't upload because of github error

### DISCUSSION THREAD

**Discussion by suku1515** [community/community#58738] [GitHub Education]
### Hi everyone,

I sent a certificate of enrollment in English to authenticate as a Github student. However, I couldn't solve it because of two problems and an error in Github. The reason is: You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

However, here, looking at the picture, the original school name is MokpoYudal Middle School (mokpoyudal.ms.jne.kr), but it is clearly different from Mokpo High School, so
![제발](https://github.com/community/community/assets/136946245/6275f0f9-0d89-45e3-b137-0be3a9d166b6)
 it is not certified. What should I do? Please help me

**Comment by raku2368** [comment]
I am also having a hard time registering for Github Student Developer Pack in Korea.
Although our school is a fairly large school with more than 1,000 students, our school name does not appear in the selection window and does not provide school e-mail to students, so we tried to certify students six times with a report card issued by the NICE website, but all six times it was rejected because the image did not include the school name

**Reply by suku1515** [reply]
The topic we are talking about seems to be slightly different. In my case, the school name is now recognized as a different school, not my original school, so even if I put my original school name in the picture, it does not authenticate.

**Comment by samus-aran** [comment]
Hi @suku1515 ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, [including posting the same comment in multiple places](https://github.com/orgs/community/discussions/58825). As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

**Comment by Sidd-17052004** [comment]
I am also facing issue while applying for Student Developer Pack. It is saying "We require applicants of D. Y. Patil College of Engineering, Akurdi to use one of these school-issued email addresses to apply:
![Screenshot 2024-07-02 195505](https://github.com/community/community/assets/135343976/db2e3a81-862f-4e05-a8df-0eb52931d131)  

Also the college email address is not verifying. Its already in use.

Please help me with this @suku1515
